concrete/src/Logging/Handler/DatabaseHandler.php

```diff
@@ -17,7 +17,7 @@ protected function write(array $record)
             $this->initialize();
         }
 
-        $uID = ($record['extra']['user'][0])?$record['extra']['user'][0]:0;
+        $uID = ($record['extra'] && $record['extra']['user'])?$record['extra']['user'][0]:0;
```

Comment:
What about simply this?

```php
$uID = $record['extra']['user'][0] ?? 0;
```

Reply:
Yeah, I agree, that looks cleaner. Could you update it to that, @haeflimi ?

Reply:
ok it's done